packages/sku/src/services/telemetry/provider.ts

```diff
@@ -3,26 +3,42 @@ import { createRequire } from 'node:module';
 import { getAddCommand } from '../packageManager/packageManager.js';
 import banner from '../../utils/banners/banner.js';
 
+import _debug from 'debug';
+
 const require = createRequire(import.meta.url);
 
-function noop() {}
+const debug = _debug('sku:telemetry');
```

Comment:
This is the same debug log namespace used by `@seek/sku-telemetry`. So if you enable it, you'll either see the noop logs, or the real logs, but never both.